docs/requirements.txt

```diff
@@ -0,0 +1,5 @@
+mkdocs==1.5.3
```

Comment:
This file duplicates the existing `requirements-docx.txt` file in the root directory and also uses older versions (at least for MkDocs). I would propose to use the existing one for now.

Reply:
Sorry, missed this one, fixed it in [843dfb6](https://github.com/unitycatalog/unitycatalog/pull/677/commits/843dfb6e441d3e635954cdd9cbaa6427bf9e47c6)